virttest/qemu_devices/qdevices.py

```diff
@@ -991,6 +992,105 @@ def verify_unplug(self, dev_type, monitor):
         return False
 
 
+@total_ordering
+class IOThread(QObject):
+    """
+    iothread representation.
+    attributes of iothread
+    "id", "poll-max-ns", "poll-grow", "poll-shrink"]
+    """
+
+    def __init__(self, id, params=None):
+        """Constructor."""
+        if params is None:
+            params = dict()
+        params["id"] = id
+        self.id = id
+        self.__attached_devices = list()
+        kwargs = dict(backend="iothread", params=params)
+        super(IOThread, self).__init__(**kwargs)
+
+    def __len__(self):
+        """Get count of attached devices."""
+        return len(self.__attached_devices)
+
+    def __eq__(self, other):
+        """Equal."""
+        if isinstance(other, IOThread):
+            return (len(self), self.id) == (len(other), other.id)
+        return super(IOThread, self).__eq__(other)
+
+    def __lt__(self, other):
+        """Less than."""
+        return (len(self), self.id) < (len(other), other.id)
+
+    @staticmethod
+    def query(monitor):
+        """Get a dictionary of info iothreads."""
+        out = monitor.info("iothreads", debug=False)
+        if isinstance(monitor, qemu_monitor.HumanMonitor):
```

Comment:
Please use query-iothreads in QMP instead, which is supported officially.

Reply:
is `QMP` or `HMP` determined by the caller of `verify_unplug`?

Reply:
misunderstood it, no problem now, thanks.